### DISCUSSION TITLE
Dev logger error using bun

### DISCUSSION THREAD

**Discussion by 4350pChris** [vikejs/vike#1221] [Discussion]
### Description

Vike version: `0.4.144`
Bun version: `1.0.7`

I'm encountering an error message in my Vue app using express without further debug info.

By inserting `console.trace` in `utils/assert` in my node_modules folder I could get the stack trace to show me that the line `assert(httpRequestId === store.httpRequestId);` in `loggerNotProd.js` seems to be the perpetrator.

Here's an entire stack trace (after the express handlers and having removed my private paths):
```
at assert (/node_modules/vike/dist/esm/utils/assert.js:54:4)
      at getCategory (/node_modules/vike/dist/esm/node/plugin/shared/loggerNotProd.js:210:12)
      at logRuntimeInfo (/node_modules/vike/dist/esm/node/plugin/shared/loggerNotProd.js:32:21)
      at logHttpRequest (/node_modules/vike/dist/esm/node/runtime/renderPage.js:191:4)
      at /node_modules/vike/dist/esm/node/runtime/renderPage.js:44:4
      at renderPage (/node_modules/vike/dist/esm/node/runtime/renderPage.js:33:26)
```

Another issue is that most requests hang indefinitely, leading to a timeout after 40 seconds. After that happens, every request will lead to the above stack trace.

I'm not sure if this is even reproducible, if I find the time I'll try to do it.

**Comment by brillout** [comment]
Moving to GitHub Discussion as these are most likely issues with Bun.

**Comment by brillout** [comment]
@4350pChris If you can't reproduce the same issues with Node.js, then it's most likely an issue with Bun and not Vike. That said, PR welcome to accomodate things for Bun.
